### DISCUSSION TITLE
积分礼物更新（3.11）

### DISCUSSION THREAD

**Discussion by itcodes** [xitu/gold-miner#8198] [Show and tell]
> 如何兑换礼物：微信私聊管理员 **@chnyifan**，发送 **`GitHub 用户名 + 礼物名称和数量 + 姓名 + 电话 + 收货地址`** 即可。啥？没有管理员微信？赶紧微信搜索 `chnyifan` 添加吧，注意备注 **`译者礼物兑换`**。
>
> 没有想要的礼物？[联系管理员](https://github.com/itcodes) 添加吧。
> TODO：兑换礼物采用提交 issue 来解决，同时机器人能记录下来积分变动。

| 掘金周边礼物 |
| :-: |
| ![](https://i.loli.net/2018/09/12/5b98bbbc2bcdc.png) | 
| 掘金纪念币/虚拟比特币：2 积分（不单发）

| 掘金 徽章 | 掘金 徽章 |
| :-: | :-: |
| ![徽章](https://user-images.githubusercontent.com/8282645/110794711-fc14ab00-82b0-11eb-8c52-d8cc5e926e01.png) | ![徽章](https://user-images.githubusercontent.com/8282645/110794707-fa4ae780-82b0-11eb-96a2-224ed59f5b75.png) |
| 掘金 全套徽章：25 积分 | 掘金  徽章 ：5 积分（不单发） |

| 掘金 周边 | 掘金 周边 |
| :-: | :-: |
| ![鼠标垫](https://user-images.githubusercontent.com/8282645/110794428-b5bf4c00-82b0-11eb-92dc-98a4d479b5bf.png) | ![卫衣](https://user-images.githubusercontent.com/8282645/110794037-434e6c00-82b0-11eb-9f59-a58e648e8ff9.png) |
| 掘金 鼠标垫：20 积分 | 掘金  卫衣 ：45 积分 |
| ![掘金帆布袋](https://user-images.githubusercontent.com/8282645/110793815-02eeee00-82b0-11eb-9ac5-6ee6e9ce69fa.png) | ![掘金搪瓷杯](https://user-images.githubusercontent.com/8282645/110793803-fff3fd80-82af-11eb-8ac6-bad35190c3c1.png) |
| 掘金 手提包：20 积分 | [掘金 搪瓷杯] 15 积分 |

| Google 赞助礼物 | Google 赞助礼物 |
| :-: | :-: |
| ![](https://user-images.githubusercontent.com/1147941/64185888-784e5700-cea0-11e9-9a15-78bc1524cecd.png) | ![](https://user-images.githubusercontent.com/1147941/64185948-96b45280-cea0-11e9-95ea-086f99c8bc21.png) |
| Android 猪年手办：15 积分 | Android 月饼手办：15 积分 |
| ![](https://user-images.githubusercontent.com/1147941/64185694-24dc0900-cea0-11e9-8571-7bcead301fe6.png) | ![Google音响](https://user-images.githubusercontent.com/1147941/64185003-f90c5380-ce9e-11e9-92d2-a73e978d0ac3.png) |
| Google 手提包：15 积分（无货） | [Google DIY 纸板音响](https://www.kickstarter.com/projects/axelpfaender/berlin-boombox) 45 积分（无货） |

| Google 赞助礼物（采购中） |掘金 礼物 |
| :-: |:-: |
| ![](https://trello-attachments.s3.amazonaws.com/5b98c9fc4eb3362db5475dfc/800x400/982e2359b4de90d2fde48522a8d52c67/WechatIMG368.jpeg.jpg) |![](https://user-images.githubusercontent.com/8282645/110799205-e6ee4b00-82b5-11eb-9fa3-f5245c60b388.png)
| Google 鲁班锁（加袋子）：17 积分 | apple home pod mini：150 积分 

| 礼物 |
| :-: |
| ![Kindle PaperWhite 4](https://user-images.githubusercontent.com/1147941/64177010-2d791300-ce91-11e9-9d29-67eac8030612.jpg)|
| [Kindle PaperWhite 4](https://www.amazon.com/All-new-Kindle-Paperwhite-Waterproof-Storage/dp/B07CXG6C9W)：200 积分 |

| 礼物 | 礼物 |
| :-: | :-: |
|![octoplush](https://user-images.githubusercontent.com/1147941/64185148-3375f080-ce9f-11e9-9700-d3971b30a91b.png) |![小米台灯](https://user-images.githubusercontent.com/1147941/64185398-ad0dde80-ce9f-11e9-88dc-4f04cda682f0.png) |
| [GitHub Octoplush 玩偶：40 积分](http://github.myshopify.com/products/octoplush) | [米家 LED 智能台灯](https://www.mi.com/mjsmartlamp/)：35 积分 |

| 礼物 | 礼物 |
| :-: | :-: |
| ![树莓派](https://user-images.githubusercontent.com/8282645/110797254-dc32b680-82b3-11eb-8a78-292bf1acd6e4.png) | ![大 Enter 按键](https://user-images.githubusercontent.com/1147941/64185743-3f15e700-cea0-11e9-8e45-cd51cdf3c26b.png) |
| 树莓派 4b（4G版） 官方套餐：80 积分 | 大 Enter 键：10 积分 |

| 礼物 | 礼物 |
| :-: | :-: |
| ![](https://user-images.githubusercontent.com/8282645/110797043-9f66bf80-82b3-11eb-966d-424133a0902d.png)| ![](https://user-images.githubusercontent.com/8282645/110797057-a1c91980-82b3-11eb-9007-e655d296e14a.png) |
| K2 机械键盘：80积分 | K8机械键盘：90积分 |

| 礼物 | 礼物 |
| :-: | :-: |
| ![](http://ww1.sinaimg.cn/large/006tNc79gw1fa0vyf8hlmj30dw0dwabd.jpg)| ![](http://ww1.sinaimg.cn/large/006tNc79gw1fa0vyf8hlmj30dw0dwabd.jpg) |
| 未完待续 | 未完待续 |

**Comment by PassionPenguin** [comment]
赶紧上线处理礼品兑换。
